### PR TITLE
Sequence edit view

### PR DESCRIPTION
**Pull request by Kr0nox**

### CODE REVIEW

**Comment by Kr0nox** [comment]
Except for a bug with editing with formatting selected text after it was set with `quill.setConent(...)`, this PR is ready for review.
For the sake of getting this feature implemented an tested I would suggest moving this bug into the bug finding phase

File: frontend/src/components/contentDisplay/DesignOptions.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  'blue',
+  'orange',
+  'yellow',
+  'purple',
+  'white',
+];
+export const defaultTextColor = 'black';
+export const sizes = ['10px', '14px', '18px', '32px', '64px'];
+export const defaultTextSize = '14px';
+export const fontFamilies = [
```
Comment (c0derMo) [inline frontend/src/components/contentDisplay/DesignOptions.ts:14]: I'd suggest when we go over the fonts to make them cross-platform, we also check these again

File: frontend/src/components/sequenceEditOptionsTab/ImageOptionsTab.vue [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  'update-content',
+]);
+
+const refContent = ref(props.imageContent);
+
+/**
+ * Opens a file dialog to select a new image.
+ */
+function selectImage(): void {
+  let input = document.createElement('input');
```
Comment (c0derMo) [inline frontend/src/components/sequenceEditOptionsTab/ImageOptionsTab.vue:52]: Can we limit the filesize for this? We don't want teachers to upload 40gb 4k images

File: model/slide/content/TextContent.ts [hunk 3]
```diff
@@ -1,15 +1,10 @@
 import { Content } from './Content';
-import { TextSnippet } from './TextSnippet';
```
Comment (c0derMo) [inline model/slide/content/TextContent.ts:2]: We can also entirely remove the TextSnippet file

**Review by c0derMo — CHANGES_REQUESTED** [review]
minor things

File: frontend/src/components/contentDisplay/DesignOptions.ts (hunk 1 above)
Comment (Kr0nox) [reply frontend/src/components/contentDisplay/DesignOptions.ts:14]: https://www.w3schools.com/cssref/css_websafe_fonts.php

File: frontend/src/components/sequenceEditOptionsTab/ImageOptionsTab.vue (hunk 2 above)
Comment (Kr0nox) [reply frontend/src/components/sequenceEditOptionsTab/ImageOptionsTab.vue:52]: should be easily possible. what size to we want as max?

Comment (c0derMo) [reply frontend/src/components/sequenceEditOptionsTab/ImageOptionsTab.vue:52]: I'd suggest something like 2MB maybe? Should be plenty I think

**Review by a-bumblebee — APPROVED** [review]
Looks good

**Review by c0derMo — APPROVED** [review]

**Review by a-bumblebee — APPROVED** [review]

**Review by ProgramPhoenix — APPROVED** [review]

**Pull request merged**